metis-enrichment/metis-enrichment-common/src/main/java/eu/europeana/enrichment/api/external/model/EnrichmentResultBaseWrapper.java

```diff
@@ -24,7 +24,8 @@
       @XmlElement(name = "Concept", namespace = "http://www.w3.org/2004/02/skos/core#", type = Concept.class),
       @XmlElement(name = "Agent", namespace = "http://www.europeana.eu/schemas/edm/", type = Agent.class),
       @XmlElement(name = "Place", namespace = "http://www.europeana.eu/schemas/edm/", type = Place.class),
-      @XmlElement(name = "Timespan", namespace = "http://www.europeana.eu/schemas/edm/", type = Timespan.class)})
+      @XmlElement(name = "Timespan", namespace = "http://www.europeana.eu/schemas/edm/", type = Timespan.class),
+      @XmlElement(name = "Organization", namespace = "http://www.europeana.eu/schemas/edm/", type = Organization.class)})
```

Comment:
One more thing. I think this is the wrong namespace? I think this is defined in FOAF, not EDM

Reply:
Yes indeed!